.github/workflows/ci.yml

```diff
@@ -0,0 +1,97 @@
+name: CI
+
+on:
+  pull_request:
+  schedule:
+    - cron: "0 0 * * *"
```

Comment:
Short question: Does this schedule cron only for master branch? Or also for all other branches?
If the latter, we need to add a filter for only master. For all repos.